### DISCUSSION TITLE
Avoid starting GRPC server automatically

### DISCUSSION THREAD

**Discussion by softshipper** [quarkusio/quarkus#26102] [Q&A] [answered]
Hi all,
I am trying to create gRPC client in Quarkus that exchanges data with a gRPC server. When I start the Quarkus application it starts in addition automatically a gRPC server.  However, I do not want that a gRPC server get started in Quarkus application automatically. How to prevent it? 

Thanks

[a comment by an automated account is not shown]

**Comment by michalszynkiewicz** [comment, marked as answer]
The solution is to add the following property:
```
quarkus.grpc.dev-mode.force-server-start=false
```

I see we don't have it documented but I'm on PTO now. Could you create a pull request to add it to the docs, or an issue for it?
